daemon/containerd/fake_service_test.go

```diff
@@ -288,3 +290,8 @@ func (s *memoryLabelStore) Update(dgst digest.Digest, update map[string]string)
 
 	return labels, nil
 }
+
+var testImageActions = sync.OnceValue(func() metrics.LabeledTimer {
+	ns := metrics.NewNamespace("test", "containerd", nil)
+	return ns.NewLabeledTimer("image_action", "Image actions", "action")
+})
```

Comment:
I started to wonder; how much sense would it make to move all those timers to a  `metrics` package, that would have some `sync.OnceValue` to get them? (instead of making these a field in the daemon).

Not sure if it's better, or if it would be _worse_. Mostly looking that some of the initialization is a bit sprinkled throughout, which means that some namespaces are registered multiple times in different locations, e.g. there's still one in `daemon/events`; https://github.com/moby/moby/blob/a72026acbbdfcf90f0ba203abd4e0943e3d546e7/daemon/events/metrics.go#L10-L15


But I guess that won't work for this specific case if we want separate counters for testing